Eatery/Network/NetworkManager.swift

```diff
@@ -111,6 +111,29 @@ struct NetworkManager {
                     diningItems[dateString] = allMenuItems
                     eventItems[dateString] = eventsDictionary
                 }
+                
+                var swipeDataPoints = [SwipeDataPoint]()
+                for swipeDatum in eatery.swipeData {
+                    guard let swipeDatum = swipeDatum else { continue }
+                    
+                    let swipeDatumStartDate = self.timeDateFormatter.date(from: String(swipeDatum.startTime))
+
+                    let swipeDatumEndDate = self.timeDateFormatter.date(from: swipeDatum.endTime)
+                    guard swipeDatumStartDate != nil && swipeDatumEndDate != nil else { continue }
+                    let swipeDatumStartHour = Calendar.current.component(.hour, from: swipeDatumStartDate!)
+                    let swipeDatumStartMinute = Calendar.current.component(.minute, from: swipeDatumStartDate!)
+                    var swipeDatumEndMinute = Calendar.current.component(.minute, from: swipeDatumEndDate!)
+                    swipeDatumEndMinute = (swipeDatumEndMinute == 0) ? 60 : swipeDatumEndMinute
```

Comment:
I feel this invariant should be enforced somewhere other than when we parse it, perhaps inside `SwipeDataPoint`. I had always assumed that `minute` was between 0 and 59.